Cargo.toml

```
@@ -12,10 +12,10 @@ readme = "README.md"
 exclude = ["assets/", "docs/assets"]
 
 [dependencies]
-glam = { version = ">=0.21, <=0.29" }
-hashbrown = "0.14.5"
+glam = { version = ">=0.21, <=0.31" }
```

I think I would rather only allow up to 0.30 since 0.31 is not released yet and could potentially not be compatible.
```suggestion
glam = { version = ">=0.21, <=0.30" }
```

---

My bad, I somehow missed  equal sign in the lower or equal constraint.